ffw/BasicCommunicationRPC.js

```diff
@@ -748,7 +748,8 @@ FFW.BasicCommunication = FFW.RPCObserver
           'minute' : current_time.getUTCMinutes(),
           'hour' : current_time.getUTCHours(),
           'day' : current_time.getUTCDate(),
-          'month' : current_time.getUTCMonth(),
+          // Month should be in 1..12 according to HMI_API
```

Comment:
@AKalinich-Luxoft Please specify that this is according to APPLINK-21928. Is any other functionality of the SDL affected by this change?